lib/beacon/router.ex

```diff
@@ -153,6 +156,19 @@ defmodule Beacon.Router do
     end
   end
 
+  @doc """
+  Creates a beacon sitemap index.
+  """
+  defmacro beacon_sitemap_index(filename) do
+    quote bind_quoted: binding(), location: :keep do
```

Comment:
I think we should start using `generated: true` as well for injected code (the other macro is missing it but I think it should have it as well).

```suggestion
    quote bind_quoted: binding(), location: :keep, generated: true do
```